airflow/models/dagrun.py

```diff
@@ -1080,43 +1096,50 @@ def _create_task_instances(
             # TODO[HA]: We probably need to savepoint this so we can keep the transaction alive.
             session.rollback()
 
-    def _revise_mapped_task_indexes(self, task: MappedOperator, session: Session) -> Iterable[TI]:
-        """Check if task increased or reduced in length and handle appropriately"""
+    def _revise_map_indexes_if_mapped(self, task: Operator, *, session: Session) -> Iterator[TI]:
+        """Check if task increased or reduced in length and handle appropriately.
+
+        Currently missing tis are created and returned if possible. Expansion
```

Comment:
```suggestion
        Missing tis are created and returned if possible. Expansion
```

"Currently missing" made me question whether you meant "currently, missing tis are..." or "currently missing tis are...". Just saying "missing tis" does the job I think.